contracts/native/version-control/src/util.rs

```diff
@@ -0,0 +1,33 @@
+use crate::{contract::VCResult, error::VCError};
+
+use cosmwasm_std::{Addr, BankMsg, Coin, CosmosMsg, MessageInfo};
+
+pub fn validate_native_funds(
+    msg_info: MessageInfo,
+    fee: Coin,
+    receiver: Option<Addr>,
+) -> VCResult<Vec<CosmosMsg>> {
```

Comment:
This should probably not return a Vec as there is no scenario in which it will return +1 msgs. Option is a better fit. 

also this functions does more than just validate the native funds so maybe that can be reflected in its name?